src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -25,47 +25,115 @@
 
   public static final String HDFS_URL_CONFIG = "hdfs.url";
   private static final String HDFS_URL_DOC = "HDFS connection URL.";
-  public static final String RECORD_WRITER_PROVIDER_CLASS_CONFIG = "record.writer.provider.class";
-  private static final String RECORD_WRITER_PROVIDER_CLASS_DOC = "";
+
+  public static final String FORMAT_CONFIG = "format";
+  private static final String FORMAT_DOC = "The format class to use when writing data to HDFS.";
+  public static final String FORMAT_DEFAULT = "io.confluent.connect.hdfs.avro.AvroFormat";
```

Comment:
Are we choosing this as a default for some reason? Does it provide the best support of features, or is it an arbitrary choice?